TUTORIAL.md

```diff
@@ -356,32 +383,32 @@ Ready to go?
+You will also see an entry under your slave, in a numbered row (probably #1) called **Building part of flowname #10**.
 
 Why are there two executors consumed by one flow build?
-Every flow build itself runs on the master, using a _flyweight executor_: an uncounted slot that is assumed to not take any significant computational power.
+Every flow build itself runs on the master, using a **flyweight executor**: an uncounted slot that is assumed to not take any significant computational power.
 This executor represents the actual Groovy script, which almost all of the time is idle, waiting for a step to complete.
 Flyweight executors are always available.
 
 When you run a `node` step, a regular heavyweight executor is allocated on a node (usually a slave) matching the label expression, as soon as one is available.
 This executor represents the real work being done on the node.
 If you start a second build of the flow while the first is still paused with the one available executor, you will see both flow builds running on master.
-But only the first will have grabbed the one available executor on the slave; the other _part of flowname #11_ will be shown in _Build Queue (1)_.
-(After a moment the console log for the second build will note that it is still waiting for an available executor.)
+But only the first will have grabbed the one available executor on the slave; the other **part of flowname #11** will be shown in **Build Queue (1)**.
+(shortly after, the console log for the second build will note that it is still waiting for an available executor).
 
-To finish up, click the ▾ beside either executor entry for any running flow and select _Paused for Input_, then click _Proceed_.
-(You can also click the link in the console output.)
+To finish up, click the ▾ beside either executor entry for any running flow and select **Paused for Input**, then click **Proceed**
+(you can also click the link in the console output).
 
-## Workspaces
+## Allocating Workspaces
 
-Besides waiting to allocate an executor on a node, the `node` step also automatically allocates a _workspace_: a directory specific to this job where you can check out sources, run commands, and do other work.
-Workspaces are _locked_ for the duration of the step: only one build at a time can use a given workspace.
-So if multiple builds need a workspace on the same node, additional workspaces will be allocated.
+In addition to waiting to allocate an executor on a node, the `node` step also automatically allocates a **workspace**: a directory specific to this job - where you can check out sources, run commands, and do other work.
+Workspaces are locked for the duration of the step: only one build at a time can use a given workspace.
+If multiple builds need a workspace on the same node, additional workspaces will be allocated.
 
-_Configure_ your slave, and set _# of executors_ to 2 (and _Save_).
+**Configure** your slave, set **# of executors** to two and **Save**.
```

Comment:
Probably better to leave as 2 since that is what you actually need to type here.